### DISCUSSION TITLE
FYI: kernel 6.1.0 + overlayfs2 is broken

### DISCUSSION THREAD

**Discussion by mattico** [AbdBarho/stable-diffusion-webui-docker#456] [General]
`RUN --mount` directories' parents cannot be deleted subsequently when using overlayfs2 on kernel 6.1.0.
See: https://github.com/moby/moby/issues/45487

This is a problem for things like:

https://github.com/AbdBarho/stable-diffusion-webui-docker/blob/d47e77f19c82411d0e1a32f6926c94478bdd1eea/services/AUTOMATIC1111/entrypoint.sh#L24

I solved this by downgrading the kernel or switching the docker daemon storage driver.

**Comment by AbdBarho** [comment]
Thank you for reporting this, I subscribed to the issue, hopefully it gets fixed soon.
